fem/ceed/solvers/full-assembly.hpp

```diff
@@ -12,15 +12,15 @@
 #ifndef MFEM_CEED_ASSEMBLE_HPP
 #define MFEM_CEED_ASSEMBLE_HPP
 
-#include "../../config/config.hpp"
+#include "../interface/ceed.hpp"
 
 #ifdef MFEM_USE_CEED
-#include <ceed.h>
-#include "../../linalg/sparsemat.hpp"
 
 namespace mfem
 {
 
+class SparseMatrix;
```

Comment:
Very minor: maybe just put `class SparseMatrix` inline in `CeedOperatorFullAssemble` since it's only used once.

Reply:
Done in [5239c10](https://github.com/mfem/mfem/pull/2952/commits/5239c10c1cc324f0af20f82e68b2925da2840969)